.github/workflows/ci.yml

```diff
@@ -181,11 +181,11 @@ jobs:
         uses: actions/checkout@v3
       # using bundler as the test updater
```

Comment:
this comment should be deleted now that the implementation detail is buried within `updater/script/build`